projects/components/form-field/src/form-field.component.ts

```diff
@@ -229,11 +232,15 @@ export const PS_FORM_FIELD_CONFIG = new InjectionToken<PsFormFieldConfig>('PS_FO
 })
 export class PsFormFieldComponent implements OnChanges, AfterContentInit, OnDestroy {
   @Input() public createLabel = true;
-  @Input() public floatLabel: FloatLabelType = 'auto';
+  @Input() public floatLabel: FloatLabelType = this.matDefaults?.floatLabel || 'auto';
   @Input() public hint: string = null;
-  @Input() public appearance: MatFormFieldAppearance = 'legacy';
+  @Input() public appearance: MatFormFieldAppearance = this.matDefaults?.appearance || 'legacy';
   @Input() public subscriptType: PsFormFieldSubscriptType = this.defaults ? this.defaults.subscriptType : 'resize';
   @Input() public hintToggle: boolean | null = null;
+  @Input() public hideRequiredMarker: boolean =
+    this.matDefaults?.hideRequiredMarker !== null && this.matDefaults?.hideRequiredMarker !== undefined
```

Comment:
Setting hideRequiredMarker of matDefaults duplicates a feature of the psFormService tryDetectRequired. And it could possibly contradict it if I set tryDetectRequired = true and hideRequiredMarker = true also. I assume there are cases to override this for single form-fields but there should not be two options to achieve the same goal

Reply:
I don't think that these two settings are ment to achieve the same goal, since tryDetectRequired sets the required attribute of the formControl and hideRequiremarker only sets the visibility of the asteric in the label of the formField.
see https://github.com/angular/components/blob/1e5c18cc075016f3b86efd1c67a4b213ade042dc/src/material/form-field/form-field.html#L52